package.json

```diff
@@ -400,6 +400,7 @@
     "test-unit-watch:cljs": "yarn concurrently -n 'cljs,tests' 'yarn build-watch:cljs' 'yarn test-unit-keep-cljs --watch'",
     "test-visual": "yarn build && ./bin/build-for-test && yarn test-visual-run",
     "test-visual:loki": "yarn loki test --chromeFlags='--headless --disable-gpu'",
+    "test-visual:loki-approve-diff": "ls .loki/difference | xargs -I _ find .loki/current -name _ | xargs -I _ cp _ .loki/reference/",
```

Comment:
When running loki with `--storiesFilter`, `yarn loki approve` seems to copy over what's generated in `current` and remove what's not there. This makes it a bit harder to select which file I want to add to the commit.

This command only update snapshots that generate diff.